server/src/index.ts

```diff
@@ -264,6 +264,20 @@ io.on('connection', async connection => {
 				}, 10000);
 			});
 
+			connection.on('editCaptainName', ({ id, name }) => {
+				const currentCaptain = selectedMatch.captains.get(id)!
+
+				selectedMatch.captains.set(id, {
```

Comment:
Is there a reason why you're doing this instead of using `currentCaptain.name = name`